src/OpenLoco/Entities/EntityManager.h

```diff
@@ -12,12 +12,13 @@ namespace OpenLoco::Vehicles
 
 namespace OpenLoco::EntityManager
 {
-    constexpr size_t numEntityLists = 6;
+    constexpr size_t numEntityLists = 7;
     constexpr size_t maxEntities = 20000;
 
     enum class EntityListType
     {
         null,
+        null2,
```

Comment:
Could you add comment denoting which of these lists is used for what entities?

Reply:
yeah at the time i didn't know but i can now say its the null money entities

Reply:
Yeah, I asked because I'd seen something to that end on Discord. :) Thanks